packages/default/scss/table/_layout.scss

```diff
@@ -0,0 +1,241 @@
+            border-style: solid;
+            box-sizing: border-box;
+            display: flex;
+            min-height: auto;
+            vertical-align: top;
+
+            td,
+            span {
+                text-align: left;
+                font-weight: bold;
+                white-space: nowrap;
+                text-overflow: ellipsis;
+                overflow: hidden;
+            }
+        }
+    }
+
+    // Footer
+    .k-table-footer {
+        width: 100%;
+        max-width: none;
+        border-width: 1px 0 0 0;
+        border-style: solid;
+        display: table;
+        table-layout: fixed;
+        empty-cells: show;
+        outline: none;
+
+        .k-table-footer-wrap {
+            width: 100%;
+            box-sizing: border-box;
+            display: flex;
+
+            .k-table-footer-item {
+                padding: $kendo-table-padding-y $kendo-table-padding-x;
+                text-align: left;
+                font-weight: bold;
+                white-space: nowrap;
+                text-overflow: ellipsis;
+                overflow: hidden;
+            }
+        }
+    }
+
+
+    // RTL
+    .k-rtl,
+    [dir="rtl"] {
+        &.k-table,
+        .k-table {
+            border-right-width: 0;
+            border-left-width: $grid-cell-vertical-border-width;
+        }
+
+        &.k-table .k-table-header .k-table-header-wrap {
+            border-left-width: 1px;
+            border-right-width: 0;
+        }
+
+        &.k-table .k-table-header .k-table-header-wrap .k-table-header-item,
+        .k-table .k-table-header .k-table-header-wrap  .k-table-header-item {
+            border-left-width: 0;
+            border-right-width: 1px;
+            text-align: right;
+
+            &:first-child {
+                border-right: 0;
+            }
+        }
+
+        &.k-table .k-table-body .k-table-row .k-table-cell,
+        .k-table .k-table-body .k-table-row .k-table-cell,
+        &.k-table .k-table-list .k-table-row .k-table-cell,
+        .k-table .k-table-list .k-table-row .k-table-cell {
+            border-width: 0 1px $grid-cell-horizontal-border-width 0;
+
+            &:first-child {
+                border-right: 0;
+            }
+        }
+
+        &.k-table .k-table-header .k-table-header-wrap .k-table-header-item:first-child,
+        .k-table .k-table-header .k-table-header-wrap .k-table-header-item:first-child,
+        .k-table .k-table-header .k-table-header-wrap > .k-table-cell:first-child {
+            border-right-width: 0;
+        }
+
+        &.k-table .k-table-header .k-table-group-header,
+        .k-table .k-table-header .k-table-group-header {
+            border-width: 1px 0 0 1px;
+        }
+    }
+
+    @each $size, $size-props in $kendo-table-sizes {
+        $_font-size: map-get( $size-props, font-size);
+        $_padding-x: map-get( $size-props, padding-x);
+        $_padding-y: map-get( $size-props, padding-y);
+
+        .k-table-#{$size} {
```

Comment:
Since most of the code below does the same thing (applies the same padding), can we probably group it together? Something like:

```
.k-table-#{$size} {
    font-size: $_font-size;
    
    .k-table-header .k-table-header-wrap .k-table-header-item,
    .k-table-body .k-table-row .k-table-cell,
    .k-table-list .k-table-row .k-table-cell,
    .k-table-header .k-table-group-header > td,
    .k-table-body .k-table-group-body,
    .k-table-list .k-table-group-body
    .k-table-footer .k-table-footer-wrap .k-table-footer-item {
        padding: $_padding-y $_padding-x;
    }

}
```